include/core/parameters.h

```diff
@@ -1616,22 +1616,50 @@ namespace Parameters
     parse_parameters(ParameterHandler &prm);
   };
 
-
   /**
-   * @brief Return the tensor of entry @p entry_string. If the entry is specified
-   * in the parameter file, then the changed value is returned, otherwise the
-   * default value is returned. If the entry is not equivalent to a Tensor<1,3>,
-   * an error will be thrown. This function can be use for Point<3> variables.
+   * @brief Return the tensor of entry @p entry_string. If the entry is
+   * specified in the parameter file, then the changed value is returned,
+   * otherwise the default value is returned. It can be used for 2 ou 3
+   * dimensions parameters that require a tensor of dimension 3 in any case.
+   * If the entry do not have 2 or 3 values in the list, an error is throw.
+   * This function can be use for Point<3> variables.
    *
-   * @param prm A parameter handler which is currently used to parse the simulation information
-   * @param entry_string A declare string in the parameter file.
+   * @param[in,out] prm A parameter handler which is currently used to parse the
+   * simulation information.
+   * @param[in] entry_string A declare string in the parameter file.
    *
    * @return A tensor<1,3> corresponding to the entry_string in the prm file.
    */
   Tensor<1, 3>
   entry_string_to_tensor3(ParameterHandler  &prm,
                           const std::string &entry_string);
 
+  /**
+   * @brief Return the tensor of entry @p entry_string. But it can allow handle
+   * the usage of depreciated parameters that used to be 3 individual entries
+   * instead of a list of value.
+   *
+   * Note: It does not throw an error if the new entry is not equivalent to a
+   * tensor.
+   *
+   * @param[in,out] prm A parameter handler which is currently used to parse the
+   * simulation information.
+   * @param[in] entry_string A declare string in the parameter file.
+   * @param[in] entry_string_1 A declare string in the parameter file that is
+   * the equivalent of the second double depreciated parameter of the new list
+   * parameter.
+   * @param[in] entry_string_2 A declare string in the parameter file that is
+   * the equivalent of the second double depreciated parameter of the new list
+   * parameter.
+   *
+   * @return A tensor<1,3> corresponding to the entry_string in the prm file.
+   */
```

Comment:
```suggestion
  /**
   * @brief Return the tensor of entry @p entry_string. But it can allow for
   * the usage of deprecated parameters that used to be 3 individual entries
   * instead of a list of values.
   *
   * Note: It does not throw an error if the new entry is not equivalent to a
   * tensor.
   *
   * @param[in,out] prm A parameter handler which is currently used to parse the
   * simulation information.
   * @param[in] entry_string A declare string in the parameter file.
   * @param[in] entry_string_1 A declare string in the parameter file that is
   * the equivalent of the second double deprecated parameter of the new list
   * parameter.
   * @param[in] entry_string_2 A declare string in the parameter file that is
   * the equivalent of the second double deprecated parameter of the new list
   * parameter.
   *
   * @return A tensor<1,3> corresponding to the entry_string in the prm file.
   */
```